kornia/geometry/transform/pyramid.py

```diff
@@ -410,3 +418,26 @@ def build_laplacian_pyramid(
         laplacian_pyramid.append(laplacian)
     laplacian_pyramid.append(gaussian_pyramid[-1])
     return laplacian_pyramid
+
+
+def upscale_double(x: Tensor) -> Tensor:
+    r"""Upscale image by the factor of 2, even indices maps to original indices.
```

Comment:
add function here to the docs: https://github.com/kornia/kornia/blob/master/docs/source/geometry.transform.rst